release-notes.txt

```diff
@@ -282,6 +282,7 @@ Fixed Issues from the last release (2018.10)
 #10739: sock_dns: Security issues (including remote code execution)
 #10701: gnrc: packet space in gnrc packet buffer is not released on very heavy load
 #10672: gnrc: runs full on very heavy load
+#10671: nanocoap: fix confirmable retry countdown
```

Comment:
This is a PR, not an issue. Here the list only contains issues. You may want to add this above in the file, maybe [here](https://github.com/RIOT-OS/RIOT/pull/10871/files#diff-acce5952fb2ed7b16765d3ddab1e2403R54)